src/color-picker/color-picker.css

```diff
@@ -4,6 +4,13 @@
 	gap: 1em;
 }
 
+#space_picker {
+	flex: 0 1 100%; /* occupy the entire row */
+	padding: initial;
```

Comment:
???

Reply:
Taking into account the following comment, I change it. This was just for alignment purposes.